docs/installation.rst

```diff
@@ -18,8 +18,8 @@ section in order to match versions that are compatible.
 .. _docker_install:
 
 Using pre-installed docker images
---------------------------------
-Download `here <https://www.eprosima.com/index.php/downloads-all>`_ the Micro XRCE-DDS and Fast-DDS Suite docker image that contains a pre-installed client and agent as well as some compiled examples.
+---------------------------------
+Download `here <https://www.eprosima.com/index.php/downloads-all>`__ the Micro XRCE-DDS and Fast-DDS Suite docker image that contains a pre-installed client and agent as well as some compiled examples. More information about this Docker image can be found :ref:`here <eprosima_docker_image>`.
```

Comment:
In general, i like to break lines after every period. That way, future modifications on a sentences only show as one line diff in git. This suggestion applies to all the modifications in the PR
```suggestion
Download `here <https://www.eprosima.com/index.php/downloads-all>`__ the Micro XRCE-DDS and Fast-DDS Suite docker image that contains a pre-installed client and agent as well as some compiled examples.
More information about this Docker image can be found :ref:`here <eprosima_docker_image>`.
```